docs/DeveloperGuide.md

```diff
@@ -450,23 +449,35 @@ Use case ends.
 **MSS**
 
 1.  User requests to clear all persons.
-2.  AddressBook prompts the user to confirm the action.
-3.  User confirms the action.
-4.  AddressBook clears all persons.
+2.  AddressBook clears all persons.
```

Comment:
A last step should be added where ab3 confirms the clearance.

currently, ab3 displays `Address book has been cleared!` after the clear command has been excecuted.

Reply:
Currently the dg tracks current behavior, but this is a good point. Should we add this as a feature? Would be good to say as a new user I want to confirm whether I clear contacts so that I can absolutely make sure I want to clear before I make destructive changes

Reply:
updated dg to reflect this